OpenMCSphericalShellFunctions/OpenMC_xml.py

```diff
@@ -0,0 +1,123 @@
+    S_3 = outside_sphere_1 & inside_sphere_2 #filled with specified material
+
+   # Mapping materials to geometry:
+    Void = openmc.Cell(fill=None, region = inside_sphere_1)
+    Shell = openmc.Cell(fill=M_1, region=S_3)
+    Cells = [Void, Shell]
+    geometry = openmc.Geometry(Cells)
+    geometry.export_to_xml()
+    return geometry, Void, Shell, Cells
+    
+# Source distribution:
+#Point-source is hard-coded, particle energy is a required input    
+def make_source(energy):
+    PointSource = openmc.stats.Point(xyz=(0.0, 0.0, 0.0))
+    Prob = openmc.stats.Discrete(energy, 1.0)
+    return PointSource, Prob
+
+# Assign simulation settings
+def settings(PointSource, Prob):
+    sets = openmc.Settings()
+    sets.batches = 10
+    sets.inactive = 1
+    sets.particles = 10000
+    sets.source = openmc.Source(space=PointSource, energy=Prob, strength = 1.0, particle = 'neutron')
+    sets.run_mode = 'fixed source'
+    sets.export_to_xml()
+    return sets
+
+# Define tallies
+def tallies(Shell):
+    neutron_tally = openmc.Tally(name="Neutron tally")
+    neutron_tally.scores = ['flux', 'elastic', 'absorption']
+    
+    # Implementing filter for neutron tally through W shell
+    cell_filter = openmc.CellFilter([Shell])
+    neutron_tally.filters = [cell_filter]
+
+    # Creating a tally to get the flux energy spectrum.
+    # An energy filter is created to assign to the flux tally.
+    energy_filter_flux = openmc.EnergyFilter.from_group_structure("VITAMIN-J-175")
+
+    spectrum_tally = openmc.Tally(name="Flux spectrum")
+    # Implementing energy and cell filters for flux spectrum tally
+    spectrum_tally.filters = [energy_filter_flux, cell_filter]
+    spectrum_tally.scores = ['flux']
+    
+    tall = openmc.Tallies([neutron_tally, spectrum_tally])
+    tall.export_to_xml()
+    return tall, neutron_tally, spectrum_tally
+
+def plot_universe(Void, Shell, Cells):
+    universe_ss = openmc.Universe(cells=Cells)
+    Plot = universe_ss.plot(width=(2500.0, 2500.0), basis='xz',
+              colors={Void: 'blue', Shell: 'red'}, legend=True)
+    Plot.figure.savefig('Universe')
+    return universe_ss
+
+# Exporting materials, geometry, and tallies to .xml
+def export_to_xml(element, inner_radius, outer_radius, energy):
+    OpenMC_Materials = make_material(element)
+    OpenMC_Geometry = make_spherical_shell(inner_radius, outer_radius, OpenMC_Materials)
+    OpenMC_Energy = make_source(energy)
+    OpenMC_Settings = settings(*OpenMC_Energy)
+    OpenMC_Tallies = tallies(OpenMC_Geometry[2])
+    OpenMC_Universe = plot_universe(OpenMC_Geometry[1], OpenMC_Geometry[2], OpenMC_Geometry[3])
+    #return OpenMC_Materials, OpenMC_Geometry, OpenMC_Energy, OpenMC_Settings, OpenMC_Tallies, OpenMC_Universe
+    return OpenMC_Materials, *OpenMC_Geometry, *OpenMC_Energy, OpenMC_Settings, *OpenMC_Tallies
+
+M_1, geometry, Void, Shell, Cells, PointSource, Prob, sets, tall, neutron_tally, spectrum_tally = export_to_xml(E_1, R_1, R_2, eg)

```

Comment:
I'm not sure what you do with all of these returned things, but also suggest something like this.... 
```suggestion
if __name__ == "__main__":
    args = define_parser()
    fp = args.filepath
    E_1 = args.element
    R_1 = args.inner_radius
    R_2 = args.outer_radius
    eg = args.energy

    with open(""+fp+"") as ALARA_Lib:
        Lib_Lines = ALARA_Lib.readlines()
    
    M_1, geometry, Void, Shell, Cells, PointSource, Prob, sets, tall, neutron_tally, spectrum_tally = export_to_xml(E_1, R_1, R_2, eg)

```